.github/scripts/get-component-release-notes.js

```diff
@@ -1,3 +1,37 @@
+
+function getComponentName(name){
```

Comment:
I think we need a readme for this workflow, so that we can add it to Component onboarding steps. I believe this needs to be updated for every new component?

Reply:
yea that was my initial idea, but then how often do we have new components onboarded 🤔 . In case a component is onboarded I was thinking if I can make the changes as when required. Does that sound good?

Reply:
my 2c it's called "tight coupling" ...

Reply:
so the initial design automatically took the component names as mentioned in the tracker and posted the details about them. But then if we want some cleaner looking release notes(properly grouped) I couldn't think of a different approach. Please let me know if we could go about using the older approach or use the new one.

Reply:
How does this `component_name: release_notes_url)` work when we have multiple sub-components. E.g for workbench we have notebook-controller and kf-notebook-controller

Reply:
@VaishnaviHire any thoughts on this?